ultra/competition/competition_bundle/terms_and_conditions.html

```diff
@@ -0,0 +1,3 @@
+<h2>Terms and Conditions</h2>
+
+<p>Fill in your terms and conditions here.</p>
```

Comment:
Flagging what @JenishPatel99 said so that the competition is not hosted without this being filled.